src/planwise/component/engine.clj

```diff
@@ -83,17 +83,18 @@
    (coverage-for-new-provider coverage project change {}))
   ([coverage project {:keys [id location] :as change} geom-cache]
    (let [project-id  (:id project)
-         raster-path (new-provider-coverage-raster-path project-id id)
+         raster-path (when (= (:type-of-source-demand project) "raster")
+                       (new-provider-coverage-raster-path project-id id))
          cached-geom (get geom-cache id)]
      (if (and (some? cached-geom) (.exists (io/as-file raster-path)))
```

Comment:
This breaks if `raster-path` is `nil`, which will happen for source sets of type point. In this context, a `nil` raster path should be considered as a valid.